languages/SMW_LanguageDe.php

```diff
@@ -96,8 +96,8 @@ class SMWLanguageDe extends SMWLanguage {
 		'_PVAP' => 'Erlaubt Muster',
 		'_DTITLE' => 'Anzeigetitel',
 		'_PVUC' => 'Erlaubt Wert einmal',
-		'_PEID' => 'External identifier',
-		'_PEFU' => 'External formatter uri',
+		'_PEID' => 'Kennung zur externen URI',
```

Comment:
> special property will always be used for exteral identifiers in connection with constructing an URI

Yes

`_PEID` is just reserved, the most important part is the type `_eid` which controls the behaviour and allows the output [0, 1] to be exported as:

```
<property:NDL_ID rdf:datatype="http://www.w3.org/2001/XMLSchema#string">00432003</property:NDL_ID>
<skos:exactMatch rdf:resource="https://id.ndl.go.jp/auth/ndlna/00432003"/>
``` 
Property specification follows:

```
* Has type: [[Has type::External identifier]]
* External formatter uri: [[External formatter uri::https://id.ndl.go.jp/auth/ndlna/$1]]
```


If you add `[[External formatter uri::...]]` to another property specification that is not of type `External identifier` then it just gets ignored,


[0] http://sandbox.semantic-mediawiki.org/wiki/Sp%C3%A9cial:Parcourir/Q1339
[1] http://sandbox.semantic-mediawiki.org/w/index.php?title=Sp%C3%A9cial:Export_RDF/Q1339&syntax=rdf